cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/ReplicaDistributionGoal.java

```diff
@@ -54,6 +60,8 @@
   private boolean _selfHealingDeadBrokersOnly;
   private final Set<Integer> _brokerIdsAboveBalanceUpperLimit;
   private final Set<Integer> _brokerIdsUnderBalanceLowerLimit;
+  private NavigableSet<BrokerAndSortedReplicas> _allBrokerAndReplicas;
```

Comment:
Is `_allBrokerAndReplicas` used purely for sorting the elements of `_brokerAndReplicasMap`? If so, can we indicate this via a comment? Currently it is not obvious why we don't read this variable.

Reply:
Good point. It looks we don't need this variable at all.